app/constants/ObservationsData.ts

```diff
@@ -1,19 +1,31 @@
 type Observation = {
+  // WCS columns
```

Comment:
minor thing: i understand where "columns" comes from, but it doesn't fully make sense in this place. Maybe "attributes" or something like that

Reply:
Done.